### PR TITLE
Business journey performance test

### PR DESCRIPTION
**Pull request by petechd**
### What is the context of this PR?
This adds performance investigation spike card write up, related to [this Trello card](https://trello.com/c/9FqlydjW).
### How to review 
Describe the steps required to test the changes (include screenshots if appropriate).

### CODE REVIEW

**Comment by petechd** [comment]
[Spike: Test daily benchmark using test_benchmark_business (1 day)](https://trello.com/c/9FqlydjW/5261-spike-test-daily-benchmark-using-testbenchmarkbusiness-1-day)

File: doc/performance-investigations/0019-benchmark-business-journey-performance-test/summary.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Feedback performance investigation
```
Comment (MebinAbraham) [inline doc/performance-investigations/0019-benchmark-business-journey-performance-test/summary.md:1]: Needs updating.

**Review by MebinAbraham — APPROVED** [review]
Not convinced by why session is spiking, however, since the only change between the two is schema, let's update to use business benchmark and keep an eye on it.

File: doc/performance-investigations/0019-benchmark-business-journey-performance-test/summary.md (hunk 1 above)
Comment (petechd) [reply doc/performance-investigations/0019-benchmark-business-journey-performance-test/summary.md]: Added correct title, thanks.

**Review by berroar — APPROVED** [review]

**Pull request merged**